migrations/20230915230734_tap_ravs.down.sql

```diff
@@ -1,2 +1,2 @@
-DROP TABLE IF EXISTS scalar_tap_ravs CASCADE;
-DROP TABLE IF EXISTS scalar_tap_rav_requests_failed CASCADE;
+-- DROP TABLE IF EXISTS scalar_tap_ravs CASCADE;
+-- DROP TABLE IF EXISTS scalar_tap_rav_requests_failed CASCADE;
```

Comment:
Is there any reason to remove this migration? If not, could you add it back?

Reply:
Sorry for that, failed to notice while sending pr, I'll update asap